jekyll/_cci2/about-circleci.md

```diff
@@ -11,74 +10,83 @@ version:
+**Why?** Continuous integration improves team productivity, efficiency, confidence, and happiness. Find problems and solve them quickly. Release higher quality, more stable products.
 
-As an operator or administrator of CircleCI installed on your own servers, CircleCI provides monitoring and insights into your builds and uses Nomad for scheduling, see the [CircleCI Operations Guide]({{site.baseurl}}/2.0/server-3-overview/) for complete documentation.
+![CircleCI about image]( {{ site.baseurl }}/assets/img/docs/arch.png)
 
-## Summary
-{: #summary }
+## CircleCI in your workflow
+{: #circleci-in-your-workflow}
 
-After a software repository on GitHub or Bitbucket is authorized and added as a project to [circleci.com](https://circleci.com), every code change triggers automated tests in a clean container or VM. CircleCI runs each [job]({{site.baseurl}}/2.0/glossary/#job) in a separate [container]({{site.baseurl}}/2.0/glossary/#container) or VM. That is, each time your job runs CircleCI spins up a container or VM to run the job in.
+A software repository on a supported version control system needs to be authorized and added as a project on [circleci.com](https://circleci.com). Every code change then triggers automated tests in a clean container or virtual machine. CircleCI runs each [job]({{site.baseurl}}/2.0/glossary/#job) in a separate [container]({{site.baseurl}}/2.0/glossary/#container) or [virtual machine](https://circleci.com/developer/images?imageType=machine).
 
-CircleCI then sends an email notification of success or failure after the tests complete. CircleCI also includes integrated [Slack and IRC notifications]( {{ site.baseurl }}/2.0/notifications). Code test coverage results are available from the details page for any project for which a reporting library is added.
+CircleCI then sends an email notification of success or failure after the tests complete. CircleCI also includes integrated [Slack and IRC notifications]({{site.baseurl}}/2.0/notifications). Code test coverage results are available from the details page for any project for which a reporting library is added.
 
-CircleCI may be configured to deploy code to various environments, including AWS CodeDeploy, AWS EC2 Container Service (ECS), AWS S3, Google Kubernetes Engine (GKE), Microsoft Azure, and Heroku. Other cloud service deployments are easily scripted using SSH or by installing the API client of the service with your job configuration.
+CircleCI may be configured to deploy code to various environments, including:
+- AWS CodeDeploy
+- AWS EC2 Container Service (ECS)
+- AWS S3, Google Kubernetes Engine (GKE)
+- Microsoft Azure
+- Heroku
 
-## What is continuous integration?
-{: #what-is-continuous-integration }
+Other cloud service deployments are easily scripted using SSH or by installing the API client of the service with your job configuration.
```

Comment:
Maybe we should add in that orbs are available for lots of popular platforms to help with deployment configuration too. 

I'm not 100% sure but this line about SSH ans API clients sounds like it might be out of date re how we present deployment now. I'll think about who we can ask.

Also - I know this was in here already so not your language but we try not to use "easy", "easily", "simply" for empathy/not to assume how things feel for readers etc.

Reply:
@rosieyohannan Yeah good catch -- we can probably add something about deploying with orbs.